node/service/src/lib.rs

```diff
@@ -1053,7 +1053,7 @@ where
 		if role.is_authority() { Some(keystore_container.sync_keystore()) } else { None };
 
 	// We currently only run the BEEFY gadget on the Rococo and Wococo testnets.
```

Comment:
outdated
```suggestion
```

Reply:
I think it still applies. We will only enable it if the `--beefy` flag is present and chain spec is either rococo or wococo.